src/map/pc.c

```diff
@@ -12260,14 +12260,19 @@ static void pc_scdata_received(struct map_session_data *sd)
 		pc->expire_check(sd);
 	}
 
-	if( sd->state.standalone ) {
+
+	if (sd->sc.data[SC_SOULENERGY] != NULL)
```

Comment:
nit: This seems unrelated to the commit it's in. Split into individual commit.

Reply:
There's no additional logic here, I reordered it so clif->pLoadEndAck is called last.

Reply:
Hmmm. okay. I was looking at it more in the terms of you moved SC_SOULENERGY check.